### PR TITLE
RAC-1223: [Backport] RAC-1215: fix error when attributeOption doesn't have label

### PR DESCRIPTION
**Pull request by ghoussard**
**Description (for Contributor and Core Developer)**

Cherry picked from commit 1e291ffa1a0bacf174708416e7f44b5b450a460d)

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: tests/back/Pim/Structure/Integration/AttributeOption/SqlSearchAttributeOptionsIntegration.php [hunk 1]
```diff
@@ -21,13 +21,50 @@ public function setUp(): void
...
         $this->createAttributeOptions('color', 'red', ['fr_FR' => 'Rouge', 'en_US' => 'Red'], 5);
         $this->createAttributeOptions('color', 'black', ['fr_FR' => 'Noir', 'en_US' => 'Black'], 4);
         $this->createAttributeOptions('color', 'blue', ['fr_FR' => 'Bleu', 'en_US' => 'Blue'], 3);
         $this->createAttributeOptions('color', 'brown', ['fr_FR' => 'Brun', 'en_US' => 'Brown'], 2);
         $this->createAttributeOptions('color', 'white', ['fr_FR' => 'Blanc', 'en_US' => 'White'], 1);
     }
 
+    public function test_it_should_return_all_options_without_or_without_locales(): void
+    {
+        $searchParameters = new SearchAttributeOptionsParameters();
```
Comment (StevenVAIDIE) [inline tests/back/Pim/Structure/Integration/AttributeOption/SqlSearchAttributeOptionsIntegration.php:34]: For me it didn't work => you should give the local in the parameters, when the local parameter is given the query doesn't work

**Review by StevenVAIDIE — COMMENTED** [review]
Add a CHANGELOG.md entry

File: tests/back/Pim/Structure/Integration/AttributeOption/SqlSearchAttributeOptionsIntegration.php (hunk 1 above)
Comment (ghoussard) [reply tests/back/Pim/Structure/Integration/AttributeOption/SqlSearchAttributeOptionsIntegration.php:34]: :ah: We have to fix master also

Comment (StevenVAIDIE) [reply tests/back/Pim/Structure/Integration/AttributeOption/SqlSearchAttributeOptionsIntegration.php:34]: Don't worry you can fix it when you pullup

**Review by StevenVAIDIE — APPROVED** [review]

**Review by grena — APPROVED** [review]

**Review by grena — APPROVED** [review]

**Review by grena — APPROVED** [review]

**Pull request merged**